amplify/cli.json

```diff
@@ -4,11 +4,11 @@
       "addmissingownerfields": false,
       "improvepluralization": false,
       "validatetypenamereservedwords": true,
-      "useexperimentalpipelinedtransformer": false,
+      "useexperimentalpipelinedtransformer": true,
```

Comment:
Is it automatically changed or manually changed?

Reply:
This is automatically generated.